components/content-service/pkg/storage/s3.go

```diff
@@ -117,6 +117,10 @@ func (rs *PresignedS3Storage) DeleteObject(ctx context.Context, bucket string, q
 		}
 	}
 
+	if len(objects) == 0 {
```

Comment:
I added this but not sure if it makes sense. Without this DeleteObjects will complain that it lacks input attributes when ListObjects  does not return any values. 
- If nothing needs to be deleted we can just return without an error
- On the other hand if no objects are found this could indicate an error condition because we expect that objects are there and we should return an error